templates/components/products/sales-badge.html

```diff
@@ -0,0 +1,14 @@
+{{#if badge_view '===' 'sash'}}
```

Comment:
I believe product-badge name suites better for this component

Reply:
I don't mind. changed